setup.py

```diff
@@ -84,6 +84,7 @@ def find_version(*parts):
               'nbval',
               'pytest',
               'pytest-cov',
+              'pyyaml',
```

Comment:
This is already in the regular requirements, so it doesn't need to be here.

```suggestion
```

Reply:
Whoops I missed that, fixed in d92ccd1.